src/org/exist/storage/BrokerWatchdog.java

```diff
@@ -8,10 +27,19 @@
 
 import org.exist.EXistException;
 
+/**
+ * Traces the lease of a Broker
+ *
+ * Note that tracing the stack is expensive
+ * this should only be done in debug mode by
+ * enabling the System Property {@link #TRACE_BROKERS_PROPERTY_NAME}
```

Comment:
@dizzzz Many! If you look at org.exist.util.Configuration - you will also find that many (but not all) options from conf.xml are overridable by setting System properties. Also much of the eXist jetty config is overridable with System properties (and even was before I moved us to Jetty 9). Also there is a good spattering of `System.getProperty` in other places of the eXist code base.

If you start a documentation project, I am happy to contribute some too :-)